pkg/reconciler/kubernetes/tektonconfig/extension.go

```diff
@@ -63,3 +106,13 @@ func (oe kubernetesExtension) Finalize(ctx context.Context, comp v1alpha1.Tekton
 	}
 	return nil
 }
+
+func changeNamespace(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetNamespace() != targetNamespace {
+			u.SetNamespace(targetNamespace)
+			return nil
+		}
+		return nil
+	}
+}
```

Comment:
we do have an already exiting transformer to do this, don't we .
why don't we reuse it.